Changelog automation task (#21)
Yoast/plugin-grunt-tasks: README.md

```

### The `update-changelog-with-latest-pr-texts` task
#### Using our configuration
We implement the following configuration:
- `options`

>     - The `useEditDistanceComapair` value is set to `true`.

Reviewer: ```suggestion
    - The `useEditDistanceCompare` value is set to `true`.
```

    - The `options.pluginSlug` value is  from the Grunt configuration: `pluginSlug`.
    - The `options.commitChangelog`: true,

#### Overview
In your project's Gruntfile, add a section named `update-changelog-with-latest-pr-texts` to the data object passed into `grunt.initConfig()`.
```js
grunt.initConfig( {
    update-changelog-with-latest-pr-texts: {
        options: {},     // Global options.
        taskName: {      // The name of your task.
            options: {}, // Task-specific options.
            src: {},     // Files to target.
        }
    }
} )
```

#### Options

> ##### useEditDistanceComapair

Reviewer: ```suggestion
##### useEditDistanceCompare
```

Type: `Boolean`  
Default: `false`

> Setting this to `true` allows the deletion duplicate line items with a Distance Comapair valu higher than 90.

Reviewer: ```suggestion
Setting this to `true` allows the deletion duplicate line items with a Distance Compare value higher than 90.
```

Default value: '""'

Optional value to add default entrys to a new created section.

_Note: `VERSIONNUMBER` will be replaced by a dynamic value_

##### useANewLineAfterHeader
Type: `Boolean`  
Default: `true`

Setting this will effect the format of the resulting changelog.

##### Usages Example
```js
update-changelog-with-latest-pr-texts: {
    "wordpress-seo": {
        options: {
            readmeFile: "./readme.txt",
            releaseInChangelog: /[=] \d+\.\d+(\.\d+)? =/g,
            matchChangelogHeader:  /[=]= Changelog ==\n\n/ig,
            newHeadertemplate: "== Changelog ==\n\n" +"= " + "VERSIONNUMBER" + " =\nRelease Date: " + "DATESTRING"  + "\n\n",
            matchCorrectHeader: "= " + "VERSIONNUMBER" + "(.|\\n)*?\\n(?=(\\w\+?:\\n|= \\d+[\.\\d]+ =|= Earlier versions =))",
            matchCorrectLines: "= " + "VERSIONNUMBER" + "(.|\\n)*?(?=(= \\d+[\.\\d]+ =|= Earlier versions =))",
            matchCleanedChangelog: "= " + "VERSIONNUMBER" + "(.|\\n)*= Earlier versions =",
            replaceCleanedChangelog: "= Earlier versions =",
            pluginSlug: "wordpress-seo",
            defaultChangelogEntrys: "",
            useANewLineAfterHeader: true,

>             useEditDistanceComapair: true,

Reviewer: ```suggestion
            useEditDistanceCompare: true,
```
